src/Console/Command/FixCommand.php

```diff
@@ -157,8 +157,9 @@ public function getHelp(): string
 
             * <comment>none</comment>: disables progress output;
             * <comment>dots</comment>: multiline progress output with number of files and percentage on each line.
+            * <comment>bar</comment>: single line progress output with number of files and calculated percentage.
```

Comment:
I modified 2 files to violate CS and tried to fix them.
I think first reported file should be on new line


<img width="759" alt="Screenshot 2023-12-24 at 13 24 51" src="https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/assets/2716794/ad3d54aa-a7ff-414a-8a0e-5bbce92acb74">

Reply:
Fixed, when progress gets 100% new lines are printed.